maps/DeadSpace/Ishimura_areas.dm

```diff
@@ -2,6 +2,7 @@
 /area/ishimura
 	icon = 'maps/DeadSpace/ishimura_areas.dmi'
 	ship_area = TRUE
+	forced_ambience = list('sound/ambience/dsamb_standardcorridor.ogg') //by default in the ship we play the corridor sound.
```

Comment:
Do not use forced_ambience. It will cause looping or jarring issues in relation to ambience changes.

Please simply use ambience =, and not forced_ambience =.